tutorial/local-setup.md

```diff
@@ -35,58 +41,62 @@ PDF, Man, Epub, and others, including PHP which is what we use at php.net.
 
 ### Install PhD
 ```
-$ cd /tmp
+$ cd phpdoc
 $ git clone https://github.com/php/phd.git
 $ php phd/render.php --help
 ```
 
 ### Use PhD to build the documentation
 ```
-$ cd /tmp/git/doc-en
+$ cd phpdoc
 $ php doc-base/configure.php
-$ php /tmp/phd/render.php --docbook /tmp/git/doc-en/doc-base/.manual.xml --package PHP --format php
-$ cd /tmp/git/doc-en/mydocs/php-web/
+$ php phd/render.php --docbook doc-base/.manual.xml --package PHP --format php
+$ cd output/php-web
 ```
 
-**Note:** This builds the php.net version of the documentation, but does not contain
-the files and includes used to run php.net. In other words, files like the php.net
-headers and footers are not built by PhD and are instead stored in a separate git
-module (web-php).
+PhD creates the `output` directory and builds the PHP manual files into the `output/php-web` directory.
+
+**Note:** This builds the php.net version of the documentation, but does not include the necessary files to run
+php.net. In other words, files like the php.net headers and footers are not built by PhD and are instead stored in a
+[separate git module (web-php)](https://github.com/php/web-php).
 
 Alternative: The XHTML format is simple and does not require mirroring the php.net
 website. The following builds manual pages as plain HTML files:
```

Comment:
I'd be inclined to put this *before* the `php-web` rendering.  That way, seeing the results of the first render is an easy one (just `open` a HTML file), and the `php-web` instructions can all be lumped together with the local website mirror section.

Reply:
I redid this section to explain the differences between XHTML and PHP. I hope it reads easier.